pkg/plugin/factory.go

```diff
@@ -20,6 +21,7 @@ type Resolver struct {
 	namespace          string
 	serviceAccountName string
 	client             client.Client
+	operatorConfig     etc.Config
```

Comment:
PluginFactory is used by CLI and Operator. So I'm wondering if instead of configuring this behaviour with environment variable it should to stored in the `starboard` ConfigMap that is ensured in the starboard-system namespace. WDYT?

By injecting etc.Config we implicitly make CLI aware of Go packages designed for Operator.